ext/ldap/tests/ldap_first_attribute_error.phpt

```diff
@@ -16,5 +16,6 @@ try {
     echo $e->getMessage(), "\n";
 }
 ?>
---EXPECT--
+--EXPECTF--
+Deprecated: Usage of ldap_connect with two arguments is deprecated. in %s on line %d
```

Comment:
All of these tests should be adjusted to use the non-deprecated form instead. Ideally there should be just one test for the deprecation.

Reply:
Yeah. I thought about that tonight as well after I modified all the tests. It's on my list for today...